src/theme/context-menu.ts

```diff
@@ -81,6 +85,20 @@ export default (options: Required<ModuleOptions>) => ({
       }
     }
   },
+  compoundVariants: [...(options.theme.colors || []).map((color: string) => ({
+    color,
```

Comment:
Shouldn't there be an `active: false` here? 🤔

Reply:
It works, but now I'll add a flag to make it clearer to others